### DISCUSSION TITLE
Include timestamp to output filename

### DISCUSSION THREAD

**Discussion by MarcoRossignoli** [coverlet-coverage/coverlet#1254] [Ideas]
Ref issue https://github.com/coverlet-coverage/coverlet/issues/378

Possible new option `--CoverletOutputIncludetimestamp`

cc: @daveMueller @petli @tonerdo

**Comment by tonerdo** [comment]
Coverlet is mostly invoked from a shell or within a build system where variables can be used. Since the filename is configurable, I don't see why we have to provide this out of the box. We can't possibly support all forms of naming conventions everyone wants.

**Reply by MarcoRossignoli** [reply]
True, I think the only useful scenario is where a dev run in local with "plain vanilla" commands like `dotnet test` without any vars/script. 
Also looks like we don't have too much thumbs up for this feature, I think that we can close for now.

**Comment by Bertk** [comment]


**Reply by danielmeza** [reply]
One common use case is when we use the solution inside a `dotnet test mysolution.sln`  we can specify the output parameter here but then the existing coverage result will be override by the last test project. So a way to specify the naming or a random name will fix this issue.

**Comment by Bertk** [comment]
`--results-directory` shall be used with test project specific report name instead. This is supported by coverlet.MTP package.

https://learn.microsoft.com/en-us/dotnet/core/testing/microsoft-testing-platform-test-reports#options

<img width="682" height="278" alt="image" src="https://github.com/user-attachments/assets/be894417-7959-4051-bd71-275d1eebdfb4" />
